### DISCUSSION TITLE
Pandas 帶你入門資料科學

### DISCUSSION THREAD

**Discussion by danwan1027** [1101-GS4719-C/Python-Community-GS4719-C#684] [優秀Python套件]
Pandas 是 python 的程式庫之一，被用來執行資料分析，對於需要處理大量數據的專案或資料科學家，pandas 都能幫助我們跨越一般文書處理程式無法做到的事。
這篇文章帶領我們從安裝（pip install pandas）到基礎語法操作（Series, DataFrame 等），讓入門 pandas 成為易上手的工具，藉由它敲開資料科學領域的大門。

https://medium.com/seaniap/pandas基礎介紹-進入資料科學的領域-be9894b3548

**Comment by Chiway1229** [comment]
謝謝這篇文章的分享，感覺這個套件對於資管系的我能有很大的幫助
不僅是在未來修習資料庫時可以應用
甚至之後還能利用這項技術進行數據的整理，進而發現資料中隱藏的寶藏

**Comment by a45320614** [comment]
Pandas似乎是成為資料科學家必學的套件呢
不論是整理、維護、存取甚至分析資料都能輕鬆解決!
不過我也想問一下Pandas跟MySQL能否有實際應用上的配合呢?

**Comment by Yunhans** [comment]
pandas似乎可以應用的範圍很多呢

**Comment by williamyen0623** [comment]
原來還有pandas這種更強大的資料分析工具

**Comment by manipulateed** [comment]
pandas 的語法豐富多元，能學到非常多資料前處理的方法，帶給我很大的收穫。

**Comment by Zoe06060606** [comment]
Pandas用來繪製資料分析的圖很好用！

**Comment by dabbie109303501** [comment]
之前上過的機器學習概論主要就是用這個套件，很好用

**Comment by Pietra1226** [comment]
謝謝分享，這讓我想要回去複習之前修的一堂資料處理的通識課。感覺未來很實用

**Comment by ching020721** [comment]
謝謝分享，第一次接觸pandas套件是在nlp課程上，現在也常常會需要用到，是很好的複習~

**Comment by evanchou1204** [comment]
Pandas的功能對從事資訊科學領域方面的人真的蠻好用的，也可以跟資料庫配合，好實用！

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
不管是什麼系 程式語言的學習都是不可或缺的 在不久的將來 絕對也會是一大潮流

**Comment by Kotori0501** [comment]
謝謝分享
大氣資料整理時需要利用pandas進行處理，因此有了這網址可以學習到很多基礎應用

**Comment by swerfg03joe** [comment]
有時做實驗累積很多資料，會使用python幫忙處理和分類，pandas確實是一個很好用的工具，之前還有在人工智慧的課程上用過，但最重要的是，pandas整理出來的表格還能和其他套件混用，更為方便

**Comment by HuangHsinTzu** [comment]
謝謝分享! 寒假有事情做了!!
對資料分析這塊蠻有興趣的
隨著越來越多裝置電子化
未來的資料量只會越來越多
好好利用的話 那些看似沒用的資料也可以有很驚奇的發現!

**Comment by BlahFish1006** [comment]
pandas在處理大量數據時，真的是個直覺又簡便的好工具，減少了在資料處理中許多繁瑣的流程。
